packages/shortest/src/browser/core/browser-tool.ts

```diff
@@ -440,7 +513,37 @@ export class BrowserTool extends BaseBrowserTool {
           }
         }
 
-        case "sleep": {
+        case InternalActionEnum.WAIT:
```

Comment:
This is similar to `sleep`, only that the unit of measurement here for `duration` is seconds, while for `sleep` is miliseconds. Some future refactoring can clean up and consolidate the logic.